config/taxonomy/application.values.schema.json

```diff
@@ -0,0 +1,84 @@
+{
+    "$schema": "http://json-schema.org/draft-04/schema#",
+    "$id": "application.values.schema.json",
+    "title": "Application Values Taxonomy",
+    "description": "Values contributed to by the mesh for data applications.",
+    "definitions": {
+        "intent": {"$ref": "policymanager.structs.schema.json#/definitions/intent"},
```

Comment:
These are definitions not values, but the filename indicates that it should include only values.
The definitions should be part of another scheme file, probably the in the M4DModule scheme

Reply:
Both structs and values schemas have definitions for example [here](https://github.com/IBM/the-mesh-for-data/blob/9928d662fc48085667f0d340fc2617bdf41d7fbe/config/taxonomy/module.values.schema.json#L6) in the M4D module values schema, we still use definitions. The term `definitions` just enables us to reuse these in other json schemas.